test/ex_webrtc/peer_connection_test.exs

```diff
@@ -885,6 +885,27 @@ defmodule ExWebRTC.PeerConnectionTest do
     end)
   end
 
+  describe "get_description" do
```

Comment:
Nitpick: Could we put this after set_local_description test? I tried to match the order of tests with the order of functions in peer_connection.ex for easier navigation
```suggestion
  describe "get_local_description/1" do
```